tests/tools/create_dummy_box.sh

```diff
@@ -8,15 +8,15 @@ if [ $# -ne 1 ]; then
 	exit 1
 fi
 
-if [ -f "$DIR/dummy.box" ]; then
+if [ -f "${DIR}/dummy.box" ]; then
 	echo "Box already created"
 	exit 0
 fi
 
 PROVIDER="$1"
-TMPDIR=`mktemp -d`
-cd "$TMPDIR"
+TMPDIR="$(mktemp -d)"
+cd "${TMPDIR}" || exit 1
```

Comment:
I'm not sure to understand the aim of adding '||exit 1'. We're running with 'set set -euxo pipefail` and the directory is created the line before. Same for the cd "${DIR}".

Reply:
ok. better be safe than sorry :)